ruff_cli/src/commands.rs

```diff
@@ -114,7 +114,7 @@ pub fn run(
             .unwrap_or_else(|(path, message)| {
                 if let Some(path) = &path {
                     let settings = resolver.resolve(path, pyproject_strategy);
-                    if settings.rules.enabled(&Rule::E902) {
+                    if settings.rules.enabled(&Rule::IOError) {
```

Comment:
Did you change these manually, or via some automation?

Reply:
I did the initial conversion automatically via a small Python script however that script isn't idempotent and the last commit also contained some manual changes so I have been manually rebasing since.

Reply:
(I can try to make this the next thing we merge.)

Reply:
Ok great let's do that ... I already addressed your two other comments locally ... then I'll start the final rebase ... doing it automatically this time because there have been too much changes to do this manually ... I'll have to split the last commit into two one for the manual changes and one for the automatic ones and update my python script^^.

Reply:
Ok, all good. I'll hold off on merging anything until this is updated and in.

Reply:
See the last commit message for the script I used to generate it (the last commit is now 100% autogenerated).